pytorch_lightning/overrides/distributed.py

```diff
@@ -75,3 +76,58 @@ def prepare_for_backward(model: DistributedDataParallel, output: Any):
             model.reducer.prepare_for_backward([])
     else:
         model.require_forward_param_sync = False
+
+
+# Taken from https://github.com/jpuigcerver/PyLaia/blob/master/laia/data/unpadded_distributed_sampler.py#L35
+class UnRepeatedDistributedSampler(DistributedSampler):
```

Comment:
This looks nicer to me

```suggestion
class UnrepeatedDistributedSampler(DistributedSampler):
```

Reply:
Done !